package-lock.json

```diff
@@ -4684,6 +4691,13 @@
         "cryptiles": "2.x.x",
         "hoek": "2.x.x",
         "sntp": "1.x.x"
+      },
+      "dependencies": {
+        "hoek": {
+          "version": "2.16.3",
```

Comment:
This version is vulnerable, and why is it being added here.

Reply:
See above reply.